src/smc-webapp/r_misc/icon.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   inverse?: boolean;
   Component?: JSX.Element | JSX.Element[];
   style?: any;
-  onClick?: () => void;
+  onClick?: (event?: any) => void; // TODO tighten what event could be
```

Comment:
optionally adding `event` here is necessary for one usage

Reply:
Use `event?: React.MouseEvent`. https://fettblog.eu/typescript-react/events/

Reply:
Use `event?: React.MouseEvent`. https://fettblog.eu/typescript-react/events/

Reply:
ok